src/main/java/javaClasses/homework_4/davit_balabekyan/staff/Developer.java

```diff
@@ -0,0 +1,49 @@
+package javaClasses.homework_4.davit_balabekyan.staff;
+
+import java.util.Objects;
+
+public class Developer extends Staff {
+
+    private String techStack;
+
+    public Developer(String techStack) {
```

Comment:
The Developer extends class Staff which has two other fields (name, lastName), what about instantiating them too? Within these two constructors you can not set name and last name

Reply:
Done)